thoth/sourcemanagement/__init__.py

```diff
@@ -15,4 +15,4 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-__version__ = "0.2.6"
+__version__ = "0.2.7"
```

Comment:
Please left the version bump on our bots. They will make sure the release is propagated and all there relevant bits get updated (e.g. CHANGELOG file).

Reply:
After the bot's bump it, I should upload to pypi right?

Reply:
They will do it for you.

Reply:
Okay thanks! :+1: